style.css

```diff
@@ -0,0 +1,604 @@
+body {
+	font-family: 'Gotham Pro';
+	font-size: 14px;
+	max-width: 1030px;
+	min-width: 320px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ProjectMod2_EKh/pullRequest?prid=6247910)